CHANGELOG.md

```diff
@@ -1,6 +1,277 @@
+   - Transition radiation plugin #3003
+   - Add output and python module doc for radiation plugin #3052
+   - Add reference to thesis for emittance plugin doc #3101
+   - Plugins: ADIOS & PhaseSpace Wterminate #2817
+   - Calorimeter Plugin: Document File Suffix #2800
+ - tools:
+   - Support alpaka accelerator `threads` #2701
+   - Add getter for omega and n to python module #2776
+   - Python Tools: Incorporate sim_time into readers and visualizers #2779
+   - Add PIConGPU memory calculator #2806
+   - Python visualizers as jupyter widgets #2691
+   - pic-configure: add `--force/-f` option #2901
+   - Correct target thickness #2873
+   - CMake: Warning in 3.14+ Cache List #3008
+   - Add an option to account for PML in the memory calculator #3029
+   - Update profile hemera-hzdr: CMake version #3059
+   - Travis CI: OSX sed Support #3073
+   - CMake: mark cuda 10.2 as tested #3118
+   - Avoid bash complition file path repetition #3136
+   - Bashcompletion #3069
+   - Jupyter widgets output capture #3149
+   - Docs: Add ionization prediction plot #2870
+   - pic-edit: clean cmake file cache if new param added #2904
+   - CMake: Honor _ROOT Env Hints #2891
+   - Slurm: Link stdout live #2839
+
+**Bug Fixes:**
+ - PIC:
+   - fix EveryNthCellImpl #2768
+   - Split `ParserGridDistribution` into `hpp/cpp` file #2899
+   - Add missing inline qualifiers potentially causing multiple definitions #3006
+   - fix wrong used method prefix #3114
+   - fix wrong constructor call #3117
+   - Fix calculation of omega_p for logging #3163
+   - Fix laser bug in case focus position is at the init plane #2922
+   - Fix binomial current interpolation #2838
+   - Fix particle creation if density zero #2831
+   - Avoid two slides #2774
+   - Fix warning: comparison of unsigned integer #2987
+   - Fix style issues with particlePusherAcceleration #2781
+   - Fix species.param docu (in all examples too) #2795
+   - Fix species.param example doc and grammar #2793
+ - PMacc:
+   - Typo fix in Send/receive buffer warning #2924
+   - Explicitly specify template argument for std::forward #2902
+   - Fix signed int overflow in particle migration between supercells #2989
+   - Boost 1.67.0+ Template Aliases #2908
+   - Fix multiple definitions of PMacc identifiers and aliases #3036
+   - Fix a compilation issue with ForEach lookup #2985
+ - plugins:
+   - Fix misspelled words in plugin documentation #2705
+   - Fix particle merging #2753
+   - OpenMPI: Use ROMIO for IO #2857
+   - Radiation Plugin: fix bool conditions for hdf5 output #3021
+   - CMake Modules: Update ADIOS FindModule #3116
+   - ADIOS Particle Writer: Fix timeOffset #3120
+   - openPMD: use particle id naming #3165
+   - Include int16 and uint16 types as traits for ADIOS #2929
+   - Fix observation direction of transition radiation plugin #3091
+   - Fix doc transition radiation plugin #3089
+   - Fix doc rad plugin units and factors #3113
+   - Fix wrong underline in TrasRad plugin doc #3102
```

Comment:
_TransRad_ 

I fixed the title of the pull just so that it is searchable without the typo even though the content of the pull is minor